installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -0,0 +1,4 @@
+[id="proc-config-rhdh-custom-resource_{context}"]
+= Configuring the {product-short} Custom Resource
+
+Updates to the Backstage Custom Resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
```

Comment:
```suggestion
Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
```